scale-tests/configs/2018-10-04-mwst8.env

```diff
@@ -11,6 +11,7 @@ SECURITY="strict"
 # Test configuration ###########################################################
 
 SHOULD_INSTALL_INFRASTRUCTURE=true
+SHOULD_UNINSTALL_INFRASTRUCTURE=false
```

Comment:
Is there some interaction between `SHOULD_INSTALL_INFRASTRUCTURE` and `SHOULD_UNINSTALL_INFRASTRUCTURE`?

Reply:
No, just similar name. Do you want it moved to the bottom?

Reply:
Right, if you select any other steps, then yes. Any ideas how to make this clearer? `SHOULD_UNINSTALL_INFRASTRUCTURE_AT_THE_END`?

Reply:
I was also thinking of renaming the envvar. As I mentioned, not critical though.